pandas/tests/io/test_parquet.py

```diff
@@ -712,6 +712,12 @@ def test_timestamp_nanoseconds(self, pa):
         df = pd.DataFrame({"a": pd.date_range("2017-01-01", freq="1n", periods=10)})
         check_round_trip(df, pa, write_kwargs={"version": "2.0"})
 
+    @td.skip_if_no("pyarrow", min_version="0.14")
+    def test_timezone_aware_index(self, pa):
+        idx = [datetime.datetime.now(datetime.timezone.utc)]
+        df = pd.DataFrame(index=idx)
+        check_round_trip(df, pa, write_kwargs={"version": "2.0"})
```

Comment:
Is the round tripping also working for fastparquet?

Reply:
Now it should work. I've changed it from `pytz.FixedOffset()` to `timezone(-timedelta(hours=hours, minutes=minutes))`.
Btw fastparquet round trip doesn't work with empty index name (it's read back as `'index'`).
In the tests there are workarounds without comments. Is this known/expected?

Reply:
Originally it worked for fastparquet, because `idx = [datetime.datetime.now(datetime.timezone.utc)]` used the name UTC and not +00:00 like pyarrow did (which is still a minor difference).

What didn't work is the following testcase in the PR, where it was a simple offset without a name (eg from parsing a string or constructed via `pd.DataFrame(index=[[datetime.now(timezone(timedelta(hours=0, minutes=0)))]])`